RevenueCatUI/Templates/Components/Packages/PackageGroup/PackageGroupComponentViewModel.swift

```diff
@@ -36,20 +36,42 @@ class PackageGroupComponentViewModel {
         let info = try Self.getPackages(component: component, offering: offering)
         self.defaultPackage = info.defaultPackage
 
-        self.stackComponentViewModel = try self.component.toStackComponentViewModel(
-            components: info.availablePackageComponents.map { .package($0) },
-            localizedStrings: localizedStrings,
-            offering: offering
+        let componentViewModels = try info.availablePackageComponents.map { info in
+            try PackageComponentViewModel(
+                localizedStrings: localizedStrings,
+                component: info.component,
+                offering: offering,
+                package: info.package
+            ).stackViewModel
+        }.map(PaywallComponentViewModel.stack)
+
+        self.stackViewModel = StackComponentViewModel(
+            component: .init(
+                components: [], // Empty on purpose because we are feeding this view models already
+                dimension: component.stack.dimension,
+                width: component.stack.width,
+                spacing: component.stack.spacing,
+                backgroundColor: component.stack.backgroundColor,
+                padding: component.stack.padding,
+                margin: component.stack.margin,
+                cornerRadiuses: component.stack.cornerRadiuses,
+                border: component.stack.border
+            ),
+            viewModels: componentViewModels
         )
     }
 
     static func getPackages(
         component: PaywallComponent.PackageGroupComponent,
         offering: Offering
```

Comment:
I know this wasn't changed in this PR, but maybe this name can be made a bit more precise? Something like `filterAvailablePackages()`? I'm also thinking that making it an extension on `PackageGroupComponent` could improve readability.